### PR TITLE
Color styles updates

### PR DESCRIPTION
**Pull request by TanyaEfremova**
1. Updates for color in Dark Mode:
- Info alert background
- Selected selected Safe in the drawer background.

2. Warning color styles updates
- `warning-background`
- `warning-light`
- `warning-dark`

## Checklist
* [x] I've tested the branch on mobile 📱
* [ ] I've documented how it affects the analytics (if at all) 📊
* [ ] I've written a unit/e2e test for it (if applicable) 🧑‍💻

### CODE REVIEW

**Review by katspaugh — APPROVED** [review]

**Comment by TanyaEfremova** [comment]
Following issues to be fixed: 
1. The colors for the amount of signatures collected in Transaction queue are still using the old colors;
2. The background color of the alert in [Notifications](https://design--walletweb.review-wallet-web.5afe.dev/settings/notifications?safe=eth:0xA77DE01e157f9f57C7c4A326eeE9C4874D0598b6) is still the same - need change in `color-secondary-background`

File: src/styles/vars.css [hunk 1]
```diff
@@ -26,10 +26,10 @@
   --color-info-main: #5fddff;
   --color-info-light: #b7f0ff;
   --color-info-background: #effcff;
-  --color-warning-dark: #cd674e;
+  --color-warning-dark: #C04C32;
   --color-warning-main: #ff8061;
-  --color-warning-light: #ffb7a6;
-  --color-warning-background: #fff0ed;
+  --color-warning-light: #FFBC9F;
+  --color-warning-background: #FFF1E0;
```
Comment (usame-algan) [inline src/styles/vars.css:32]: This file is auto-generated and will be overwritten. Instead you have to change the colors in `/src/components/theme/lightPalette.ts` and `/src/components/theme/darkPalette.ts` and run `yarn css-vars` to update this file.

**Comment by usame-algan** [comment]
@TanyaEfremova I updated the variables and re-generated the vars.css. Also noticed that the `secondary-background` color changed to `#1b2a22` as defined in #3066. I think we forgot to generate the file back then but the change should be reflected now.

**Comment by TanyaEfremova** [comment]
Colors fixed, thanks @usame-algan! 
The text in the conflicting transactions block is white, though, not sure, I guess it was changed by accident. 
Screenshot 2024-03-19 at 16 40 14

**Comment by usame-algan** [comment]
Thanks! I think its been there for a while. Pushed a fix nonetheless so that the color is always #121312 now

**Comment by usame-algan** [comment]
@katspaugh can you help merge the PR? It is blocked for me because it contains an unsigned commit.

**Comment by TanyaEfremova** [comment]
@usame-algan I figured the issue: the DarkPalette warning color should stay the same so far. I will create a separate PR to fix those.
Light Mode looks good!

**Pull request merged**